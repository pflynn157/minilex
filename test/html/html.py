base_path = "./base"
output_path = "./src"

# Define the keywords
keywords = [
    ("html", "Html"), ("head", "Head"), ("body", "Body"), ("title", "Title"), ("div", "Div"),
    ("h1", "H1"), ("h2", "H2"), ("h3", "H3"), ("h4", "H4"), ("h5", "H5"), ("h6", "H6")
]

# Define the symbols
symbols = [
    ("<", "TagOpen"), ("</", "CloseTagOpen"), (">", "TagClose")
]

# Define single-line comments
single_comments = []

# Define multi-line comments
multi_comments = [ ("<!--", "-->") ]
