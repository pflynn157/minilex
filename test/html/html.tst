<html>
    <head>
        <title>Hello!</title>
    </head>
    <body>
        <h1>Welcome</h1>
        <div>
            <h2>Welcome!</h2>
        </div>
    </body>
</html>
