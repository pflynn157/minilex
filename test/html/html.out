<
html
>
<
head
>
<
title
>
ID(Hello!)
</
title
>
</
head
>
<
body
>
<
h1
>
ID(Welcome)
</
h1
>
<
div
>
<
h2
>
ID(Welcome!)
</
h2
>
</
div
>
</
body
>
</
html
>

